### DISCUSSION TITLE
How can I show multiple queries in one diagram?

### DISCUSSION THREAD

**Discussion by machavo** [datalust/seq-tickets#1582] [Help]
I want to show multiple Queries in one diagram. Due to the fact that I do not have access to case, subquery , Union , CTE , How can create a that?
Currently I have these queries on separate diagrams :
Query 1 :
select count(*) as count
from stream 
where Params like '%TType25%'
group by time(5s)
limit 10000

Query 2 :
select count(*) as count
from stream 
where Params like '%TType'9%'
group by time(5s)
limit 10000

**Comment by liammclennan** [comment]
Hi @machavo,

One way is to make each query into a group. Something like:

```
select count(*) from stream
group by if Contains(Params, 'TType25') then 'TType25' else if Contains(Params, 'TType'9') then 'TType9' else 'Other', time(5s)
limit 1000
```

**Reply by machavo** [reply]
Hi @liammclennan 
Thank you for your answer, I wrote this, but this message was displayed:
"The query could not be executed.
 Syntax error (line 3, column 13): unexpected identifier `Contains`."

**Comment by liammclennan** [comment]
Which version of Seq are you using?

**Reply by machavo** [reply]
5.1.3364

**Comment by liammclennan** [comment]
Perhaps you could use some combination of `Substring` and `IndexOf` in the `group by` expression?
